### DISCUSSION TITLE
Copilot is writing a console.log bug in my vscode/emmet/JS

### DISCUSSION THREAD

**Discussion by spydmobile** [community/community#37370] [Copilot Conversations]
Copilot is amazing, and most of the time it saves me tons of time, but lately I noticed that it writes the same bug over and over (a typo) I dont know if it doing this because of my coding, or a bug in copilot or emmet or vscode - here is an example: 

when I type ```console.log(``` and hit tab, copilot will write:
```JS
console.log() 
```
it will then put the cursor in the call and offers 
```JS
console.log("something meaningful"); 
```
I hit tab to accept then enter and move on. BUT

when I type ```cl``` this triggers emmet to offer console.log
when I accept the offer it types
```JS
console.log() 
```
and places the cursor inside the call and then copilot offers:
```JS
console.log("something meaningful")); 
```
If i catch it - I am catching it almost always now, I must remove the typo and move on.
But if my eye does not catch it, this typo will break the entire javascript file.

this is perfectly consistent and repeatable/reproducible on my system.  [New Recording - 10/26/2022, 10:09:28 AM](https://vimeo.com/764224978/85439b2fa9)

**Comment by ossanna16** [comment]
Hi there @spydmobile and welcome to our community! Thank you for reporting this 🙂

**Reply by klaus-donnert** [reply]
Copilot also does this in intelliJ with braces in if statements like so:

if(whatever) {
 // when I accept the suggestion here - it adds an extra closing brace like so
}
}
